assets/js/alert.js

```diff
@@ -0,0 +1,26 @@
+const displayToast = (toast, right = '20px', opacity = '1') => {
+  toast.style.right = right;
+  toast.style.opacity = opacity;
+};
+
+const showToasts = (type,message) => {
+  const toast = document.getElementById('toast');
+  if(type === "success"){
+    toast.classList.add('text-main');
+    toast.querySelector('#icon').classList.add('bg-main');
+    toast.querySelector('#icon').innerHTML = '<i class="fas fa-check-circle"></i>';
+    toast.querySelector('span').textContent = message;
+    setTimeout(() => {
+      displayToast(toast, '-200px', '0');
+    }, 3000);
+  }else if(type === "error"){
```

Comment:
```suggestion
  } else if(type === 'error'){
```
Em lưu ý phần coding convention khi code